gamedata/movedefs.lua

```diff
@@ -57,8 +57,8 @@ local moveDatas = {
 	ABOT2 = {
 		crushstrength = 50,
 		depthmod = 0,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```

Comment:
Shouldn't this be called ABOT3 now? It seemed the name was intended to match the foorptinr size,

Reply:
i can do that at a later time
im going to soft resolve all of the naming issues later before a merge occurs